thoth/storages/graph/postgres.py

```diff
@@ -4104,7 +4092,7 @@ def _python_sync_analysis_result(
                 os_version=software_environment.os_version,
                 python_version=software_environment.python_version,
                 index_url=None,
-                sync_only_entity=True
+                sync_only_entity=document["metadata"]["arguments"]["thoth-package-extract"]["metadata"]["is_external"]
```

Comment:
`is_external`?

Reply:
mmm this is `_create_python_package_version` it does not have `is_external`